client/tailscale/tailscale.go

```diff
@@ -38,24 +38,57 @@ import (
 	"tailscale.com/tailcfg"
 )
 
-var (
-	// TailscaledSocket is the tailscaled Unix socket. It's used by the TailscaledDialer.
-	TailscaledSocket = paths.DefaultTailscaledSocket()
+// Local is the default LocalClient when using the legacy package-level functions.
+var Local LocalClient
```

Comment:
should this be a pointer?

Reply:
I'm worried about the sync.Once being copied

Reply:
https://pkg.go.dev/sync#Once says it should not be copied

Reply:
PTAL